lib/ddg-atb.js

```diff
@@ -16,7 +16,8 @@
 
 "use strict";
 
-const {Cu} = require("chrome");
+const {Cu, CC} = require("chrome");
+const XMLHttpRequest = CC("@mozilla.org/xmlextras/xmlhttprequest;1");
```

Comment:
I think this whole method of building an extension is being deprecated at some point anyway.